kubernetes/table_kubernetes_job.go

```diff
@@ -139,13 +139,45 @@ func listK8sJobs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData
 		return nil, err
 	}
 
-	jobs, err := clientset.BatchV1().Jobs("").List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return nil, err
+	input := metav1.ListOptions{
```

Comment:
Can we use name or namespace as an optional key qual in the list function? It should work if I provide any one of them.